### ISSUE TITLE
Migration Guide

### ISSUE THREAD

**Issue by KastB** [KIT-ISAS/iviz#2]
Hi,

is there some migration guide from ROS#?
Those projects seem to be quite similar, despite possible performance improvements and different visualization plugins, right?

Cheers,
Bernd

**Comment by akzeac** [comment]
Yes, they're mostly similar. I made the interface like ROS# because that's what my previous code was using.
And sure, I can write a guide in the next few days. The iviz_roslib documentation sure needs some work.
